### PR TITLE
Release/1.6.0

### PR DESCRIPTION
**Pull request by kobizz**

### CODE REVIEW

File: includes/settings/settings.php [hunk 1]
```diff
@@ -183,6 +183,15 @@ protected function create_tabs() {
 									'desc' => __( 'Elementor lets you hide the page title. This works for themes that have "h1.entry-title" selector. If your theme\'s selector is different, please enter it above.', 'elementor' ),
 								],
 							],
+							'open_images_in_lightbox' => [
+								'label' => __( 'Open Images In Lightbox', 'elementor' ),
+								'field_args' => [
+									'type' => 'checkbox',
```
Comment (KingYes) [inline includes/settings/settings.php]: Change to select control

**Review by KingYes — CHANGES_REQUESTED** [review]

File: includes/widgets/image-gallery.php [hunk 2]
```diff
@@ -21,6 +21,16 @@ public function get_categories() {
 		return [ 'general-elements' ];
 	}
 
+	public function get_script_depends() {
+		return [
```
Comment (KingYes) [inline includes/widgets/image-gallery.php]: You can write this in one line..

**Review by KingYes — CHANGES_REQUESTED** [review]

File: includes/widgets/video.php [hunk 3]
```diff
@@ -414,18 +460,17 @@ protected function _register_controls() {
 			[
 				'label' => __( 'Content Position', 'elementor' ),
 				'type' => Controls_Manager::SELECT,
-				'default' => 'center center',
 				'frontend_available' => true,
 				'options' => [
-					'center center' => __( 'Center', 'elementor' ),
-					'center top' => __( 'Top', 'elementor' ),
+					'' => __( 'Center', 'elementor' ),
```
Comment (KingYes) [inline includes/widgets/video.php]: Why the default like this?

**Review by KingYes — CHANGES_REQUESTED** [review]

File: includes/widgets/video.php (hunk 3 above)
Comment (kobizz) [inline includes/widgets/video.php]: Because of the selector

**Pull request merged**